src/pages/index.js

```diff
@@ -71,12 +74,24 @@ export default function WRMCWebsite() {
     setCurrentPage("Show Details");
   }
 
+  // determines the current and next three shows
+  const now = moment();
+
+  const upcomingShows = shows.filter(
+    (show) => (dayToInt[show.time.day] === now.day()) && (show.time.hour >= (now.hour() * 100)));
+
+  upcomingShows.sort((a, b) => compareTwoShows(a, b));
+
+  const isOnAir = upcomingShows[0].time.hour === now.hour() * 100;
+
   const placeholderPages = {
     // TODO: we should create a container component for the Home page
     // (and any other pages that end up having multiple components)
     "Home" : <div>
               <ShowOTW show={sotw} handleClick={clickShow}/> <p>{""}</p>
-              <NextThreeShows shows={allShows} handleClick={clickShow}/>
+              {isOnAir ? <DisplayCurrentShow show = {upcomingShows[0]} handleClick={clickShow}/> : <DisplayCurrentShow show = {shows.find(show => show.id === 12345)} handleClick={clickShow}/>}
```

Comment:
I think you can put the ternary operator right into the JSX, so you don't need to write out the component twice here, e.g. `<DisplayCurrentShow show={isOnAir ? upcomingShows[0] : shows.find(show => show.id === 12345)} handleClick={clickShow}/>`. This might help to reduce some clutter.